package.json

```diff
@@ -20,11 +20,7 @@
     "test:ci": "run-s test:ci:*",
     "test:dev:ava": "ava",
     "test:ci:ava": "nyc -r lcovonly -r text -r json ava",
-    "prepublishOnly": "run-s prepublishOnly:*",
-    "prepublishOnly:checkout": "git checkout master",
-    "prepublishOnly:pull": "git pull",
-    "prepublishOnly:install": "npm install",
-    "prepublishOnly:test": "npm test"
+    "prepublishOnly": "npm install && npm test"
```

Comment:
`run-s` won't work in the GitHub action since it doesn't have dependencies installed yet

Reply:
Can we add a `npm install` in the GitHub action workflow instead?

Reply:
Sure, in that case the `prepublishOnly` script would only be `npm test`?

Reply:
Maybe we could keep the tasks that are common to both manual and automated publish as `prepublishOnly`, then add an additional npm task for CI pre-publish (triggered from GitHub action)?

Reply:
See https://github.com/netlify/build/pull/2442/commits/ed3775a9bbee66efaf890bfd0e79f79b952531c0
I couldn't find a better way to do this.
I found https://github.com/ericclemmons/if-env but it fails the command when the condition is not met